### PR TITLE
5.5.0: source release preparation

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/ZSSTrMZC/16-ome-common-550-release

In preparation of the ome-common 5.5.0 release, this PR:

- bumps the `release-version` in `CMakeLists.txt`
- backports the release notes from https://www.openmicroscopy.org/announcements/ into a top-level `NEWS.md`
- add a new section for the release notes of 5.5.0 - see https://github.com/ome/ome-common-cpp/milestone/8 for a list of included PRs.

/cc @rleigh-codelibre @joshmoore

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+------------------
+
+* Added Base64 conversion functions ([#16](https://github.com/ome/ome-common-cpp/pull/16))
+* Corrected Xerces UTF8 to UTF-16 transcoding for empty strings ([#18](https://github.com/ome/ome-common-cpp/pull/18))
+* Added rankine temperature conversions ([#17](https://github.com/ome/ome-common-cpp/pull/17))
+* Added MSVC workarounds for system macros with names matching unit names ([#22](https://github.com/ome/ome-common-cpp/pull/22))
+* Removed VS2012 workarounds ([#15](https://github.com/ome/ome-common-cpp/pull/15))
+* Improved CMake configuration export ([#21](https://github.com/ome/ome-common-cpp/pull/21))
+* Added BSD licence text ([#23](https://github.com/ome/ome-common-cpp/pull/23))
+
```
Comment (rleigh-codelibre) [inline NEWS.md:56]: Extra newline here.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+5.5.0 (2017-11-27)
+------------------
+
+* Use CMP0067 to enable standard setting in feature tests ([#47](https://github.com/ome/ome-common-cpp/pull/47))
+* CMake 3.4 is the minimum version ([#48](https://github.com/ome/ome-common-cpp/pull/48))
+* Use C++11 \<thread\> and \<mutex\> ([#46](https://github.com/ome/ome-common-cpp/pull/46))
```
Comment (rleigh-codelibre) [inline NEWS.md]: Headers can be marked up verbatim.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Restore support for Boost 1.53 ([#49](https://github.com/ome/ome-common-cpp/pull/49))
+* Add support for Boost 1.65 and 1.65.1 ([#50](https://github.com/ome/ome-common-cpp/pull/50))
+* Conditionally add log and log_setup to exported configuration ([#51](https://github.com/ome/ome-common-cpp/pull/51))
```
Comment (rleigh-codelibre) [inline NEWS.md]: Should be Boost::log and Boost::log_setup (and these should be verbatim).

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+* Add documentation links ([#35](https://github.com/ome/ome-common-cpp/pull/35))
+* Add support for Boost 1.64 ([#37](https://github.com/ome/ome-common-cpp/pull/37))
+
+5.4.0 (2017-02-09)
+------------------
+
+* C++11 is now the minimum required language version, with C++14 being used when available ([#29](https://github.com/ome/ome-common-cpp/pull/29))
+* enabled the use of a restricted number of C++11 features, including enum class, nullptr, initializer lists, range-based for loops and type traits ([#31](https://github.com/ome/ome-common-cpp/pull/31))
+* enabled the use of C++11 syntax changes including “<::” not being interpreted as a trigraph and “»” being used to close nested templates instead of “> >” ([#31](https://github.com/ome/ome-common-cpp/pull/31))
```
Comment (rleigh-codelibre) [inline NEWS.md]: I think all the quoted trigraphs should be verbatim instead, since it's code.

File: NEWS.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* Google Test (gtest) is no longer built separately in each source component; the latest gtest release now allows use as a conventional library ([#28](https://github.com/ome/ome-common-cpp/pull/28))
+* source releases are now made directly from git with “git archive” ([#32](https://github.com/ome/ome-common-cpp/pull/32))
```
Comment (rleigh-codelibre) [inline NEWS.md]: "git archive" could be verbatim, since it's a command.

File: NEWS.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+* Add support for Boost 1.63 ([#30](https://github.com/ome/ome-common-cpp/pull/30))
+* Boost MPL size limits are now set with MSVC only; they are no longer required for other compilers with C++11
```
Comment (rleigh-codelibre) [inline NEWS.md]: These headers can all be verbatim.

File: NEWS.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+------------------
+
+* Document all namespaces in API reference ([#24](https://github.com/ome/ome-common-cpp/pull/24))
+
+5.3.0 (2016-07-29)
+------------------
+
+* Added Base64 conversion functions ([#16](https://github.com/ome/ome-common-cpp/pull/16))
+* Corrected Xerces UTF8 to UTF-16 transcoding for empty strings ([#18](https://github.com/ome/ome-common-cpp/pull/18))
+* Added rankine temperature conversions ([#17](https://github.com/ome/ome-common-cpp/pull/17))
```
Comment (rleigh-codelibre) [inline NEWS.md:51]: Rankine should be capitalised since it's a proper name?

File: NEWS.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+5.2.0 (2016-04-23)
+------------------
+
+* Decouple ome-common-cpp component from Bio-Formats ([#1](https://github.com/ome/ome-common-cpp/pull/1), [#9](https://github.com/ome/ome-common-cpp/pull/9))
+* Use external gtest ([#2](https://github.com/ome/ome-common-cpp/pull/2))
+* Add support for Boost 1.60 ([#4](https://github.com/ome/ome-common-cpp/pull/4))
+* Add Xalan-C support ([#7](https://github.com/ome/ome-common-cpp/pull/7))
+* Rename all uses of bioformats to ome-common ([#10](https://github.com/ome/ome-common-cpp/pull/10))
+* Add source-archive script to generate a source release ([#12](https://github.com/ome/ome-common-cpp/pull/12), [#13](https://github.com/ome/ome-common-cpp/pull/13), [#14](https://github.com/ome/ome-common-cpp/pull/14))
```
Comment (rleigh-codelibre) [inline NEWS.md]: "source-archive" should be verbatim since it's a command.

**Comment by rleigh-codelibre** [comment]
Overall looks fine.  Some comments added for some minor cosmetic markup improvements.

**Comment by sbesson** [comment]
Last commit should update the release date /cc @joshmoore

**Comment by joshmoore** [comment]
LGTM

**Comment by rleigh-codelibre** [comment]
As a general comment about the formatting of these files, I do wonder if it's necessary to put the date alongside the release version.  Could this not be moved below the section heading, before the bullet list?

**Pull request merged**